engine.py

```diff
@@ -859,7 +859,15 @@ def _set_project(self):
 
         # Since we are inserting this path into another string that will be executed in mel
         # We need to double up and backslashes.
-        proj_path = proj_path.replace("\\", "\\\\")
+        # For network paths on Windows (starting with '\\\\'), doubling backslashes is avoided
+        # as this leads to RuntimeErrors when executed in mel.
```

Comment:
I am confused between this comment and the previous one saying we need to double the \\. 
Did something change in MEL since this original comment? When was it introduced in the first place? Is it still valid?

Reply:
Removing that comment and backslash doubling for `proj_path`. Paths are already pre-escaped and `cmds.workspace` introduced in https://github.com/shotgunsoftware/tk-maya/pull/98 handles them correctly in Python, making additional escaping for MEL unnecessary.